### ISSUE TITLE
If openshift_token is used but not openshift_user|password then preflight fails

### ISSUE THREAD

**Issue by ccollicutt** [CrunchyData/postgres-operator-playbooks#8]
I see that `openshift_token` is used in the operator role, but if you add it to the inventory and do not set `openshift_user|password` then the asserts in the preflight role fail.

```sh
$ ansible-playbook -i inventory install.yml

PLAY [Deploy Crunchy PostgreSQL Operator] ****************************************************************************************************************************************************

TASK [preflight : include_tasks] *************************************************************************************************************************************************************
included: /home/centos/postgres-operator-playbooks/roles/preflight/tasks/check_openshift.yml for localhost

TASK [preflight : openshift_project should be defined] ***************************************************************************************************************************************
ok: [localhost] => {
    "changed": false, 
    "msg": "All assertions passed"
}

TASK [preflight : openshift_user should be defined] ******************************************************************************************************************************************
fatal: [localhost]: FAILED! => {
    "assertion": "openshift_user is defined and openshift_user != ''", 
    "changed": false, 
    "evaluated_to": false, 
    "msg": "Set the value of 'openshift_user' in the inventory file."
}
	to retry, use: --limit @/home/centos/postgres-operator-playbooks/install.retry

PLAY RECAP ***********************************************************************************************************************************************************************************
localhost                  : ok=2    changed=0    unreachable=0    failed=1   
```

**Comment by jasonodonnell** [comment]
I will add extra logic to handle the usage of tokens, thanks for the report @ccollicutt !

**Comment by jasonodonnell** [comment]
Fixed via #11, hope that helps @ccollicutt!

**Comment by ccollicutt** [comment]
Thanks @jasonodonnell, I will take a look in the near future but looks good to me. :) Thanks again.
